eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/publisher/impl/AlertPublisherImpl.java

```diff
@@ -199,7 +194,7 @@ private void closePlugins(List<AlertPublishPlugin> toBeClosed) {
             try {
                 p.close();
             } catch (Exception e) {
-                LOG.error(String.format("Error when close publish plugin {}!", p.getClass().getCanonicalName()), e);
+                LOG.error(String.format("Error when close publish plugin %s!", p.getClass().getCanonicalName()), e);
```

Comment:
`String.format` should use `%s` rather than `{}`

```java
@Test
public void testFormat() {
    assertEquals("a{}c", String.format("a{}c", "b"));
    assertEquals("abc", String.format("a%sc", "b"));
}
```